accounting/rewind.go

```diff
@@ -99,7 +99,13 @@ func AccountAtRound(ctx context.Context, account models.Account, round uint64, d
 		MinRound: round + 1,
 		MaxRound: account.Round,
 	}
+	ctx, cf := context.WithCancel(ctx)
```

Comment:
I think that the code is fine, but please use a different variable name for `ctx` to avoid variable shadowing.
Also, I'd suggest you'll defer the cf before the `db.Transactions` call. ( i.e. so that if the `db.Transactions` call get panicked, the context would still get destroyed )

Reply:
It's ok that you don't want to use it afterward.. the naming would just make sure the reader would understand it's a different object.

Reply:
using `defer cf()` is a little problematic because the logic would now depend on the evaluation order of defer. I think splitting the function in half should make the order clearer.

Reply:
Actually, splitting up the function forces them to be backwards. It occurs to me now that I can simply call cf multiple times.